rebasehelper/checker.py

```diff
@@ -52,29 +50,40 @@ def run_check(cls, results_dir):
         raise NotImplementedError()
 
 
-class Checker(object):
+class CheckersRunner(object):
     """
-    Class representing a process of checking final packages.
+    Class representing the process of running various checkers on final packages.
     """
 
-    def __init__(self, dir_name):
+    def __init__(self, dir_name=os.path.join(os.path.dirname(__file__), 'checkers')):
```

Comment:
You can always pass the specific path in which the checkers are placed. This is just to simplify the default case, which works. If we move the dir in our repo, we will just edit this one line. No complication from my point of view.